llmfoundry/models/mpt/configuration_mpt.py

```diff
@@ -7,6 +7,8 @@
 
 from transformers import PretrainedConfig
 
+from llmfoundry.models.utils.misc import is_torch_2_or_higher
```

Comment:
[Composer has this util](https://github.com/mosaicml/composer/blob/dev/composer/utils/misc.py#L87). Should we just import it from there?

Reply:
im ok with keeping this in foundry. minimizing composer dependencies has benefits for end users and huggingface compat.